dev/TEAL.md

```diff
@@ -189,6 +200,15 @@ these results may contain leading zero bytes.
 | `b^` | A bitwise-xor B, where A and B are byte-arrays, zero-left extended to the greater of their lengths |
 | `b~` | X with all bits inverted |
 
+The following opcodes allow for the construction and submission of
+"inner transaction"
+
```

Comment:
is the `log` opcode described anywhere?

Reply:
It has a blurb here, and further explanation in ledger.md